components/Table/Table.js

```diff
@@ -3,61 +3,95 @@ export class Table {
+      const cell = document.createElement('div');
+      cell.classList.add('table-header__cell');
+      cell.setAttribute('data-key', col.key);
+      cell.setAttribute('data-sortable', col.sortable);
+      const headerLabel = document.createElement('div');
+      headerLabel.classList.add('table-header-label');
 
-      const actionsContainer = document.createElement("div");
-      actionsContainer.classList.add("actions-container");
+      if (col.sortable) {
+        headerLabel.addEventListener('click', () => {
+          this.isOrderAsc = !this.isOrderAsc;
+          const dataKey = cell.getAttribute('data-key');
+          arrowBtn.setAttribute('asc', this.isOrderAsc);
+          this.callbackFunc(dataKey, this.isOrderAsc);
+        });
+      }
 
-      const arrowBtn = document.createElement("span");
-      arrowBtn.classList.add("material-icons");
-      arrowBtn.textContent = "south";
+      const actionsContainer = document.createElement('div');
+      actionsContainer.classList.add('actions-container');
 
-      const menuBtn = document.createElement("span");
-      menuBtn.classList.add("material-icons");
-      menuBtn.textContent = "more_vert";
+      const arrowBtn = document.createElement('span');
+      arrowBtn.classList.add('material-icons');
+      arrowBtn.textContent = 'south';
+      arrowBtn.setAttribute('name', 'arrow');
+
+      if (col.sortable) {
+        arrowBtn.addEventListener('click', () => {
+          this.isOrderAsc = !this.isOrderAsc;
+          arrowBtn.setAttribute('asc', this.isOrderAsc);
+          const dataKey = cell.getAttribute('data-key');
+          this.callbackFunc(dataKey, this.isOrderAsc);
+        });
+      }
+
+      const menuBtn = document.createElement('span');
+      menuBtn.classList.add('material-icons');
+      menuBtn.textContent = 'more_vert';
+      menuBtn.setAttribute('name', 'menuBtn');
 
       actionsContainer.append(arrowBtn, menuBtn);
-      cell.textContent = col.label;
+      headerLabel.textContent = col.label;
+      cell.append(headerLabel);
       cell.append(actionsContainer);
       return cell;
     });
     tableHeaderRow.append(...cells);
     this.tableHeader.append(tableHeaderRow);
   }
 
+  clearTable() {
+    this.tableBody.innerHTML = '';
+  }
+
   render(data) {
+    this.clearTable();
     const fragment = new DocumentFragment();
     data.forEach((element) => {
-      const rowElement = document.createElement("div");
-      rowElement.classList.add("table-row");
+      const rowElement = document.createElement('div');
+      rowElement.classList.add('table-row');
       const cells = this.columnsConfig.map((column) => {
-        const cell = document.createElement("td");
-        cell.classList.add("table-row__cell");
+        const cell = document.createElement('div');
+        cell.classList.add('table-row__cell');
         cell.textContent = element[column.key];
+        cell.setAttribute('data-key', column.key);
```

Comment:
Do we need `data-key` attribute here? It would make sense to have it on the header but not sure that we need it here